datatypes/src/main/java/org/hyperledger/besu/datatypes/TransactionType.java

```diff
@@ -63,16 +63,6 @@ public byte getEthSerializedType() {
     return (this == FRONTIER ? 0x00 : this.getSerializedType());
   }
 
-  /**
```

Comment:
I was curious about the history of this method, and found that it was introduced and used in https://github.com/hyperledger/besu/pull/2132/files#diff-f4664e850b91579cf65137f12d71d2269b6dc4d3f1c59ee5f65ae0782947809a, but then some following refactoring stopped using it, so safe to remove